src/loki/search.cc

```diff
@@ -259,9 +262,6 @@ struct bin_handler_t {
       // cache the distance
       if (distance == std::numeric_limits<float>::lowest())
         distance = node_ll.Distance(location.latlng_);
-      // the search cutoff is a hard filter so skip any outside of that
-      if (distance > location.search_cutoff_)
-        return;
```

Comment:
this was my screw up back when we added search_cutoff. should have thought about that a little harder at the time. apologies!